nav2_bringup/launch/nav2_params.yaml

```diff
@@ -140,23 +145,25 @@ world_model:
   ros__parameters:
     use_sim_time: True
 
-world_model_global_costmap:
-  ros__parameters:
-    use_sim_time: True
-    obstacle_layer:
-      enabled: False
-    always_send_full_costmap: True
-    observation_sources: scan
-    scan:
-      topic: /scan
-      max_obstacle_height: 2.0
-      clearing: True
-      marking: True
+global_costmap:
+  global_costmap:
+    ros__parameters:
+      use_sim_time: True
+      robot_radius: 0.092
```

Comment:
At risk of stating the obvious, when we rebase & merge with master, all the parameters will need to be updated to the ones from the master file, several of these are outdated and new ones have been added.

Reply:
Yes, definitely. We'll do that.